<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>b_MainWindow</class>
 <widget class="QMainWindow" name="b_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>205</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>205</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>insert</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="b_centralwidget">
   <widget class="QPushButton" name="b_pushButton_2">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>55</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>20</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>수량 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b_lineEdit">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
